packages/stream-api/src/streamApi.ts

```diff
@@ -51,30 +49,30 @@ export class StreamApi {
   public beginInteraction(): Promise<
     vertexvis.protobuf.stream.IStreamResponse
   > {
-    return this.send({
+    return this.sendRequest({
       beginInteraction: {},
     });
   }
 
   public replaceCamera({
     camera,
-  }: RecursiveRequired<
-    vertexvis.protobuf.stream.IUpdateCameraPayload
-  >): Promise<vertexvis.protobuf.stream.IStreamResponse> {
-    return this.send({
+  }: vertexvis.protobuf.stream.IUpdateCameraPayload): Promise<
+    vertexvis.protobuf.stream.IStreamResponse
+  > {
```

Comment:
With removing the `RecursiveRequired` type here, it looks like we'll lose any requirement that properties expected by FSS (like a camera with three vectors) are present. This `RecursiveRequired` type might not work with these changes due to the `frameCorrelationId` being a truly optional property, but I'm curious what your thoughts are on how/if we could continue to require properties like the camera. Seems like without some sort of enforcement here it becomes a bit more work to interact with these APIs

Reply:
I've created a ticket for improvements to Stream API. https://jira.vertexvis.net/browse/SDK-740